timemachine/fe/utils.py

```diff
@@ -145,17 +145,19 @@ def score_2d(conf, norm=2):
     return score / len(conf)
 
 
-def generate_good_rotations(mol_a, mol_b, num_rotations=3, max_rotations=1000):
+def generate_good_rotations(mol_a, mol_b, num_rotations=3, max_rotations=1000, seed=1234):
```

Comment:
nit: Be nice to fill out the type hints

Reply:
Filled out in https://github.com/proteneer/timemachine/pull/1135/commits/82dd14c543e35eeca181b38d67c3556c7b6dad30